docs/prometheus-operator/README.md

```diff
@@ -0,0 +1,388 @@
+# Prometheus operator
+
+The purpose of this document is to explain how to setup [Prometheus operator](https://github.com/coreos/prometheus-operator) for Knative observability. It also describes how to scrape metrics endpoints using service monitors.
+
+## Cluster setup
+
+Create a kubernetes cluster as mentioned [here](https://knative.dev/docs/install/knative-with-gke/#creating-a-kubernetes-cluster).
+
+> Note: A GKE cluster is used, but any other cluster type can be used as well.
+
+## Knative setup
+
+### Install Knative [Serving](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#deploy-knative-serving) and [Eventing](https://github.com/knative/eventing/blob/master/DEVELOPMENT.md#starting-eventing-controller).
```

Comment:
Did you want the users to install the monitoring components of knative/serving as well? To avoid confusion, you can suggest skipping that part.

Reply:
Done.